README.md

```diff
@@ -667,6 +667,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 * [requests](https://requests.kennethreitz.org/en/master/) - HTTP Requests for Humans™.
 * [treq](https://github.com/twisted/treq) - Python requests like API built on top of Twisted's HTTP client.
 * [urllib3](https://github.com/shazow/urllib3) - A HTTP library with thread-safe connection pooling, file post support, sanity friendly.
+* [aiosonic](https://github.com/sonic182/aiosonic) - A very fast async HTTP client
```

Comment:
Hello @sonic182,

Items are listed in alphabetical order here, so could you please put `aiosonic` to the right line.
Also, could you please set more meaningful title for this PR, e.g. `Add aiosonic`?

Thanks!

```suggestion
* [aiosonic](https://github.com/sonic182/aiosonic) - A very fast async HTTP client.
```

Reply:
Done 👍